{PROJECT_NAME}/Sources/Data/NetworkAPI/Core/RequestConfiguration.swift

```diff
@@ -2,38 +2,34 @@
 //  RequestConfiguration.swift
 //
 
-import Alamofire
 import Foundation
+import Moya
 
-protocol RequestConfiguration {
+enum RequestConfiguration {
 
-    var baseURL: String { get }
-
-    var endpoint: String { get }
-
-    var method: HTTPMethod { get }
-
-    var url: URLConvertible { get }
-
-    var parameters: Parameters? { get }
+    /// Add cases for each endpoint
+}
 
-    var encoding: ParameterEncoding { get }
+extension RequestConfiguration: TargetType {
 
-    var headers: HTTPHeaders? { get }
+    /// Return base URL for a target
```

Comment:
No need for documentation comments, it's ok to see on Moya's repository. And for other comments here.